src/schema/v1.json

```diff
@@ -1,5 +1,96 @@
 {
- "definitions": {},
+ "definitions": {
+     "artifacts": {
+         "type": "object",
+         "properties": {
+           "path": {
+             "type":"string",
+             "title":"Path",
+             "default":"",
+             "pattern":"^(.*)$"
+            },
+           "sha256": {
+             "type":"string",
+             "title":"SHA256",
+             "default":"",
+             "pattern":"[A-Fa-f0-9]{64}"
+            },
+           "size": {
+             "type":"integer",
+             "title":"Size in bytes",
+             "default": 0
+            },
+           "uncompressed-sha256": {
+             "type":"string",
+             "title":"Uncompressed SHA256",
+             "default":"",
+             "pattern":"[A-Fa-f0-9]{64}"
+            },
+           "uncompressed-size": {
+             "type":"integer",
+             "title":"Uncompressed-size",
+             "default": 0
+            }
+          },
+          "optional": [
+              "size",
+              "uncompressed-sha256",
+              "uncompressed-size"
+          ],
+          "required": [
+              "path",
+              "sha256"
+          ]
+        },
+     "image": {
+         "type": "object",
+         "required": [
+             "digest",
+             "image"
+         ],
+         "properties": {
+           "digest": {
+             "type":"string",
+             "title":"Digest",
+             "pattern":"sha256:[A-Fa-f0-9]{64}"
+            },
+           "image": {
+             "type":"string",
+             "title":"Image",
+             "pattern":"^(?!\\s*$).+"
+            }
+          }
+      },
+      "downloadable": {
```

Comment:
Slight bikeshed: technically all our artifacts are downloadable. :) Maybe something like `cloudartifact` to denote that it's hosted on a target cloud?